### PR TITLE
Members page

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
image
image
image

### CODE REVIEW

File: instances/treasury-devdao.near/widget/lib/common.jsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
-const treasuryDaoID = "${REPL_TREASURY}";
+const treasuryDaoID = "build.sputnik-dao.near";
```
Comment (Tguntenaar) [inline instances/treasury-devdao.near/widget/lib/common.jsx:1]: Is this supposed to be hardcoded?

**Review by Tguntenaar — COMMENTED** [review]
LGTM, do we want to add any tests?

File: instances/treasury-devdao.near/widget/lib/common.jsx (hunk 1 above)
Comment (Megha-Dev-19) [reply instances/treasury-devdao.near/widget/lib/common.jsx]: yeah committed it by mistake, removed it

**Comment by Megha-Dev-19** [comment]
> LGTM, do we want to add any tests?

yes, we will add it, Peter is helping out with tests, I m working on adding the pages first.

**Review by petersalomonsen — APPROVED** [review]

**Pull request merged**